### ISSUE TITLE
custom my.cnf file

### ISSUE THREAD

**Issue by ox1depl** [mmontes11/mariadb-operator#51]
Hi
I dont see any possibility how to i can use a custom my.cnf file or inject custom configuration for mariadb.
Can any one tell me how to do in this operator ?

**Comment by mmontes11** [comment]
Hey there @ox1depl !

At the moment this is currently supported, but this could be an interesting feature for the following releases. We could ispecify a custom `my.cnf` in the `MariaDB` CRD that will be mounted in the `StatefulSet` as a `Configmap`:

```yaml
apiVersion: mariadb.mmontes.io/v1alpha1
kind: MariaDB
metadata:
  name: mariadb
spec:
    ...
    myCnf: |
       [mysqld]
       bind-address=0.0.0.0
       default_storage_engine=InnoDB
       binlog_format=row
       innodb_autoinc_lock_mode=2
```

Alternatively, the used could be able to provide the `Configmap` straight away:

```yaml
apiVersion: mariadb.mmontes.io/v1alpha1
kind: MariaDB
metadata:
  name: mariadb
spec:
    ...
    myCnfConfigMapKeyRef:
        name: mariadb-config
        key: 'my.cnf'
```

Does this cover your needs?

**Comment by ox1depl** [comment]
Thanks for your reply.
You mean it's NOT supported or ...? Sorry but your answer is a bit unclear.
And yes, the possibility of mounting my.cnf as configmap would be a great option because now this operator actually provides a database that is only suitable for development environments.

**Comment by mmontes11** [comment]
Hey @ox1depl !

This feature has been released in the `v0.0.9`, make sure you update:
- https://github.com/mmontes11/mariadb-operator/releases/tag/v0.0.9
- https://github.com/mmontes11/mariadb-operator/releases/tag/helm-chart-0.9.0

Closing! Thanks for contributing to `mariadb-operator`
